Objective-J/CommonJS/lib/objective-j.js

```diff
@@ -110,6 +111,14 @@ exports.run = function(args)
                     argv.shift();
                     multipleFiles = true;
                     break
+
+                case "-c":
+                case "--objc":
+                    argv.shift();
+                    argv.unshift(require("file").path(module.path).dirname().join("objective-j/objective-c-parser.j"));
+                    argv.unshift("objj");
+
+                    return OS.system(argv);
```

Comment:
I find that weird to eat all arguments for a command, then stop everything and start another command because there is a special parameter